### ISSUE TITLE
Accessibility: Product detail review section has accessibility issues

### ISSUE THREAD

**Issue by msiewierska** [shopware/shopware#13996]
### Shopware Version

6.7.3.1

### Affected area / extension

Platform(Default)

### Actual behaviour

Product detail review section on PDP has a bunch of accessibility errors:

**Success criteria 1.3.1 (level A): review summary: fieldset is missing**

<img width="708" height="485" alt="Image" src="https://github.com/user-attachments/assets/e3ba8ff9-031c-46b6-b7ce-eaf9c3e0ae5d" />

**Success criteria 1.3.5 (level AA): review sorting select has no autocomplete attribute**

 **Success criteria 4.1.2 (level A): review summary: there are visible labels for hidden inputs**

<img width="957" height="550" alt="Image" src="https://github.com/user-attachments/assets/4e1aed26-0e16-493a-8baa-86acbdb10f5d" />

**Success criteria 1.3.1 (level A): Different rating is provided for screen reader user** 
_.product-detail-reviews-average_ element receives _ratingAverage|number_format(1)_  e.g. **4.3**

_product-review-rating-alt-text visually-hidden_ element receives
_{% set altText = 'detail.reviewAvgRatingAltText'|trans({ '%points%': points|round(1, 'floor'), '%maxPoints%': 5 }) %}_ f.e. **4.2**

meta data  <meta itemprop="ratingValue" content="4.25"> _reviews.matrix.averageRating|round(2, 'common')_  receives **4.25**

### Expected behaviour

Accessibility success criteria should be met.

### How to reproduce

Go to product page with at least several ratings. Inspect HTML, perform accessibility audit.

**Comment by MalteJanz** [comment]
@tobiasberge is this something (storefront a11y issues) you guys would like to take over or should we do it? 🙈

**Comment by MalteJanz** [comment]
@tobiasberge @Phil23 friendly ping in case you have missed my previous comment 😅

**Comment by tobiasberge** [comment]
@MalteJanz Sorry for late reply. You can take it over, if you need clarification regarding breaking changes or something please ping us. 👍

**Comment by vintagesucks** [comment]
@msiewierska Thanks for the detailed report! 🚀 

We've addressed the rating rounding mismatch for screen readers, see #15075 for details on why the other points weren't applicable.
